specter-registry/src/testmod/java/dev/spiritstudios/testmod/SpecterRegistryTestMod.java

```diff
@@ -0,0 +1,43 @@
+package dev.spiritstudios.testmod;
+
+import com.mojang.serialization.Codec;
+import net.fabricmc.api.ModInitializer;
+
+import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback;
+import net.fabricmc.fabric.api.command.v2.ArgumentTypeRegistry;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.command.argument.IdentifierArgumentType;
+import net.minecraft.command.argument.serialize.ConstantArgumentSerializer;
+import net.minecraft.network.codec.PacketCodecs;
+import net.minecraft.registry.Registries;
+
+import dev.spiritstudios.specter.api.registry.attachment.Attachment;
+import net.minecraft.util.Identifier;
+
+public class SpecterRegistryTestMod implements ModInitializer {
+
+	public static final String MOD_ID = "specter-registry-testmod";
+	public static final Identifier ATTACHMENT_ID = Identifier.of(MOD_ID, "attachment_test");
+	public static Attachment<Block, Integer> TEST_ATTACHMENT;
+
+	@Override
+	public void onInitialize() {
+		ArgumentTypeRegistry.registerArgumentType(Identifier.of(MOD_ID, "registry-id"), SpecterRegistryTestCommand.RegistryArgumentType.class, ConstantArgumentSerializer.of(IdentifierArgumentType::identifier));
```

Comment:
These might be useful to add to the main API (maybe just the `attachmentid` one)